package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@regru/webpack-config",
-  "version": "2.1.0",
+  "version": "3.0.2",
```

Comment:
bump version in package.json pls.

and why is there a major bump? any breaking changes?